mteb/tasks/Retrieval/law/GerDaLIRRetrieval.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class GerDaLIR(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="GerDaLIR",
```

Comment:
```suggestion
class GerDaLIRSmall(AbsTaskRetrieval):
    metadata = TaskMetadata(
        name="GerDaLIRSmall",
```
Could you also rename this file?